### DISCUSSION TITLE
Welcome to astro-starter Discussions!

### DISCUSSION THREAD

**Discussion by zanhk** [majesticooss/astro-starter#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by deployn** [comment]
I just wanted to say thank you. 

I already tried to use Astros which is the only template with a working i18n integration (while the dependency there seems to be deprecated).

Do you plan to migrate to the native i18n implementation, which is nearly finished at the moment?

**Reply by zanhk** [reply]
Yes, I will migrate all templates once released
